docs/metrics.md

```diff
@@ -0,0 +1,174 @@
+# METRICS
+
+## Data Analysis
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do users install and run this?
+
+What is the overall engagement of the Containers experiment?
+**This is the standard Daily Active User (DAU) and Monthly Active User (MAU) analysis.**
+
+This captures data from the users who have the add-on installed, regardless of
+whether they are actively interacting with it.
+
+![](images/kpi-1.png)
+
+### Immediate Questions
+
+* Do people use the containers feature?
+  * Click to create new container tab
+* Do people who use the containers feature continue to use it?
+  * Retention: opening a second container tab (second tab in the same container, or a tab in a second container?)
+* What containers do people use?
+  * Container name (should we record the names of custom container created by users?)
+    * \+ Number of tabs in the container (when should we measure this? on every tab open?)
+* Do people edit their containers? (do we care about editing built-in containers vs. user-created containers?)
```

Comment:
I don't think that distinction matters.  Would recording it help you answer any questions you have?